packages/js/product-editor/src/blocks/input/block.json

```diff
@@ -0,0 +1,41 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-input-field",
+	"title": "Product input control",
+	"category": "woocommerce",
+	"description": "A input control for use in the product editor.",
+	"keywords": [ "products", "input" ],
+	"textdomain": "default",
+	"attributes": {
+		"label": {
+			"type": "string",
+			"__experimentalRole": "content"
+		},
+		"property": {
+			"type": "string"
+		},
```

Comment:
I think we should support both properties on a product and metadata, to be flexible.